renovate.json

```diff
@@ -12,8 +12,8 @@
   "rollbackPrs": true,
   "packageRules": [
     {
-      "description": "Automerge all yarn & prettier changes",
-      "matchPackageNames": ["prettier", "yarn"],
+      "description": "Automerge all yarn, prettier, and gh-cli changes",
```

Comment:
I might suggest just calling this something like "automerge frequent updates" since we can see the package names easily anywy

Reply:
I think yarn/prettier don't fall into the frequent category, more something like "stable boring infra"?